src/main/scala/cromwell/binding/WdlBinding.scala

```diff
@@ -115,6 +114,11 @@ object WdlBinding {
     }
   }
 
+  /* All MemberAccess ASTs that are not contained in other MemberAccess ASTs */
+  def findTopLevelMemberAccesses(expr: AstNode) = WdlBinding.findAstsWithTrail(expr, "MemberAccess").filter {
+    case (k, v) => !(v map {case a:Ast => a.getName} contains "MemberAccess")
```

Comment:
The most opinionated official style guide I know of is the one which was recently blasted for being written by people  "who don't understand scala" :) Most of them don't really say a whole lot.

I'd argue that akka and scalaz are written by the most prominent scala developers out there. Obviously scala itself, but they admit that they're doing a lot of things other people shouldn't in order to empower the other people to not have to go that route. These codebases are actively developed by a majority of the scala developers that people should be looking up to.

However last night I took a cruise through both codebases and the lesson learned is that clearly neither are zealous about any of this sort of stuff as one could find examples for both sides of just about every stylistic debate we've had to date (ok, except long closures for HOFs). You're much more Captain Consistency than I am, I'm willing to drop a whole slew of these little discussions in the name of "hey, it's written by different people" if you are.

I didn't think about it last night but I'd also throw in stuff like cats and spire as being written by someone worthy of high praise.